package.json

```diff
@@ -83,12 +83,12 @@
     "webpack": "^5.75.0"
   },
   "peerDependencies": {
-    "@ember/test-helpers": "^2.9.3",
-    "ember-source": ">=3.28",
+    "@ember/test-helpers": ">=3.0.3",
```

Comment:
just making notes for release,
we're dropping support for:
- @ember/test-helpers < 3.0.3
- ember-source < 4
- node < 16
- `this.element` -- folks should use `qunit.dom` or `[data-test-whatever]` and `find` 
- `this.render` -- `await render` is easy find and replace

Reply:
for `this.render`, folks would need to remember to `async function (assert)` + `await render`

Reply:
Yup, something to call out in the changelog